src/main/java/uk/ac/cam/cl/dtg/segue/dao/AbstractPgDataManager.java

```diff
@@ -44,24 +49,45 @@ protected void setValueHelper(final PreparedStatement pst, final int index, fina
       pst.setString(index, ((Enum<?>) value).name());
     }
 
-    if (value instanceof String) {
-      pst.setString(index, (String) value);
+    if (value instanceof String stringValue) {
+      pst.setString(index, stringValue);
+    }
+
+    if (value instanceof Integer integerValue) {
+      pst.setInt(index, integerValue);
     }
 
-    if (value instanceof Integer) {
-      pst.setInt(index, (Integer) value);
+    if (value instanceof Long longValue) {
+      pst.setLong(index, longValue);
     }
 
-    if (value instanceof Long) {
-      pst.setLong(index, (Long) value);
```

Comment:
These look like they're changing the data type in the prepared statement from what was used previously?? Won't that clash with the db schema?

Reply:
Just double-checked this and none of the values in the database should be being changed here?
I'm not sure why the difference window is displaying it like this but, unless I'm mistaken, the only one that's _actually_ being changed is `(value instanceof java.util.Date)` becoming `(value instanceof java.util.Date)`.
Even in that case, it's still being stored in the database as an sql Timestamp.